### DISCUSSION TITLE
Question: How does the explicit search work?

### DISCUSSION THREAD

**Discussion by Deekerman** [RandomNinjaAtk/docker-lidarr-extended#64] [Q&A] [answered]
So I know it will pull songs that have the explicit tag on them vs the clean, but what if there is no explicit or clean tag? Will it download just standard albums that arent tagged explicit or clean?

Thanks

**Comment by RandomNinjaAtk** [comment, marked as answer]
All albums are either tagged as explicit or clean that I'm aware of because its a true or false statement. I don't believe the situation your alluding to exists...

**Reply by Deekerman** [reply]
So, then does that mean if an album has no explicit tracks on it it will ignore that album because its deemed clean? 

Lets take Led Zeppelin lll for example. it has no explicit lyrics. Does that mean that Deemix will ignore that album?

If that is the case, i should have my search set to both rather than Explicit.

**Reply by Deekerman** [reply]
After looking a bit, it seems that is the case. My system didnt download zep from deemix but rather from usenet. But it is on deezer. 

That makes sense, just not how i thought it worked.

**Reply by RandomNinjaAtk** [reply]
You likely set the `audioLyricType` setting as explicit, you probably want to set it to both, make sure you read the description carefully:

Description:
"both or explicit or clean :: both, is explicit preferred matching,  explicit is explicit releases only matching and clean is clean releases  only matching"
Reference: https://github.com/RandomNinjaAtk/docker-lidarr-extended#parameters

**Reply by Deekerman** [reply]
That is exactly it, yes. 

Apparently i need to update my lidarr even tho i just installed it. that will provide better search it seems

**Reply by RandomNinjaAtk** [reply]
> After looking a bit, it seems that is the case. My system didnt download zep from deemix but rather from usenet. But it is on deezer.
> 
> That makes sense, just not how i thought it worked.

You can use both the built in DL clients and traditional methods, such as usenet normally, however, which ever one wins the race and notifies lidarr to import it first, will be the one you end up with...

**Reply by Deekerman** [reply]
@RandomNinjaAtk i know this is off discussion, but once the deemix script runs through once, on start up it just looks for new artists added (i think). is there anyway to force a recheck of all of the artists in the collection through deemix or no?

**Reply by RandomNinjaAtk** [reply]
It should auto-recheck anything not found every 90 days, wiping logs folder data /config/extended/logs/folders will cause it to do it immediately

**Reply by Deekerman** [reply]
Thank you.
